src/components/Plans/PlanPaymentDialog.js

```diff
@@ -145,6 +146,7 @@ const PlanPaymentDialog = ({
   }
 
   function showPayment () {
+    onOpen && onOpen()
```

Comment:
```suggestion
   if(onOpen) onOpen()
```
Sometimes it's hard to understand where expression starts/ends when it's written like this. Better to clearly indicate  single line instruction with an `if` operator